josm.openstreetmap.de.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: josm.openstreetmap.de
+branch: stable
+tags:
+  - osm
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk10
```

Comment:
It doesn't work with openjdk11?

Reply:
ah. dunno.
Let me try.

Reply:
it seems to work. I've updated the file.